src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -660,20 +660,26 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, const pal
     }
 
     pal::string_t retval;
+    bool cli_version_specified = false;
+	bool cli_version_found = false;
+	pal::string_t cli_version;
```

Comment:
There's whitespace/indentation inconsistency here and elsewhere. (I won't comment everywhere).